src/nvim/mbyte.c

```diff
@@ -2261,7 +2261,7 @@ char_u *enc_locale(void)
 // versions).
 // Returns (void *)-1 if failed.
 // (should return iconv_t, but that causes problems with prototypes).
-void *my_iconv_open(char_u *to, char_u *from)
+void *my_iconv_open(const char_u *to, const char_u *from)
```

Comment:
could be changed to `char`, all callsites are casting to `char_u` for no reason

Reply:
You're right, I should've checked the callsites. I didn't do it because I know theres `char_u` to `char` changes going on in the repo, but in this case it's certainly a benefit. Fixed.